addons/binding/org.openhab.binding.elerotransmitterstick/src/main/java/org/openhab/binding/elerotransmitterstick/stick/DelayedCommand.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.elerotransmitterstick.stick;
+
+import java.util.concurrent.Delayed;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * @author Volker Bier - Initial contribution
+ */
+public class DelayedCommand extends Command {
+    private final long origin;
+    private final long delay;
+
+    public DelayedCommand(CommandType cmd, long delayInMillis, int priority, Integer... channels) {
+        super(cmd, channels);
+
+        delay = delayInMillis;
+        this.origin = System.currentTimeMillis();
+        this.priority = priority;
```

Comment:
Some general code improvement. priority is a field in the super class. It's better to set it via a constructor. So create a protected constructor in the super class and pass the priority value. That way the field priority doesn't have to be protected and even can be final.